### PR TITLE
Final fixes

### PR DESCRIPTION
**Pull request by BlakeDonn**
### What does this PR do?

* Polishes readme, adds final GIF
*  Fixes bug in report

### CODE REVIEW

**Review by timkeresey — APPROVED** [review]

File: src/components/Report/Report.test.tsx [hunk 1]
```diff
@@ -99,10 +99,10 @@ describe('Report', () => {
...
     expect(screen.getByText('Dream Home')).toBeInTheDocument();
     expect(screen.getByText('My Numbers')).toBeInTheDocument();
-    expect(screen.getByText('Property tax')).toBeInTheDocument();
+    expect(screen.getByText('Ten year goal')).toBeInTheDocument();
     expect(screen.getByText('Downpayment %')).toBeInTheDocument();
-    expect(screen.getByText('Mortgage term')).toBeInTheDocument();
-    expect(screen.getByText('Credit Score')).toBeInTheDocument();
+    expect(screen.getByText('Downpayment saved')).toBeInTheDocument();
+    expect(screen.getByText('Mortgage Term')).toBeInTheDocument();
     expect(screen.getByText('Report')).toBeInTheDocument();
```
Comment (Atos20) [inline src/components/Report/Report.test.tsx:106]: thanks for updating these test.

File: src/components/Report/Report.tsx [hunk 2]
```diff
@@ -1,6 +1,5 @@
-//imports
 import React, {useContext } from 'react'
-import {ReportContext} from '../../types'
+import {ReportContext, AnswerContext} from '../../types'
```
Comment (Atos20) [inline src/components/Report/Report.tsx:2]: Didn't notice we were not using the AnswersContext

File: src/components/Report/Report.tsx [hunk 3]
```diff
@@ -45,15 +46,15 @@ const Report = () => {
           categoryMainNumber={
             userReport[data].monthly_principal && `$${userReport[data].monthly_principal}`
             || userReport[data].zipcode
-            || userReport[data].downpayment_percent_saved
+            || userReport[data].downpayment_percent_saved && `%${userReport[data].downpayment_percent_saved}`
```
Comment (Atos20) [inline src/components/Report/Report.tsx:49]: is this the same as saying if this `userReport[data].downpayment_percent_saved` exits then do this `%${userReport[data].downpayment_percent_saved}`????

File: src/components/Report/Report.tsx [hunk 4]
```diff
@@ -115,49 +116,48 @@ const Report = () => {
...
               <div className="num-ci-box-up">
-                <h1 className="num-ci-title">Credit Score</h1>
+                <h1 className="num-ci-title">Mortgage Term</h1>
               </div>
 
               <Spring
                 config={{delay: 100, duration: 1000}}
                 from={{ number: 0 }}
-                to={{ number: 30 }}>
+                to={{ number: userAnswers.mortgage_term }}>
```
Comment (Atos20) [inline src/components/Report/Report.tsx:143]: thanks for updating this number

**Review by Atos20 — APPROVED** [review]

**Pull request merged**